### ISSUE TITLE
Physicians Info Page Sucks

### ISSUE THREAD

**Issue by justinrporter** [SaturdayNeighborhoodHealthClinic/snhc-web#36]
@mokkaralam @javaheriank you guys should decide what this needs to look like, and I'll help get the page up and running.

**Comment by ecd25** [comment]
I only added a survey link. It would be better to embed the survey in the page.

**Comment by justinrporter** [comment]
We should chat about this on monday. There is a good way to make changes on github, and the default kind of sucks so we should talk about it.

JRP

**Comment by ecd25** [comment]
D :

Sorry!

I hope I didn't actually make things harder for you...

On Sat, Jan 2, 2016 at 9:48 PM, Justin R. Porter notifications@github.com
wrote:

> We should chat about this on monday. There is a good way to make changes
> on github, and the default kind of sucks so we should talk about it.
> 
> JRP
> 
> > On Jan 2, 2016, at 5:31 PM, Emily Davis notifications@github.com
> > wrote:
> > 
> > I only added a survey link. It would be better to embed the survey in
> > the page.
> > 
> > —
> > Reply to this email directly or view it on GitHub <
> > https://github.com/SaturdayNeighborhoodHealthClinic/snhc-web/issues/36#issuecomment-168438467
> > .
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/SaturdayNeighborhoodHealthClinic/snhc-web/issues/36#issuecomment-168451084
> .

## 

Emily Davis

Washington University School of Medicine
Class of 2018

Cell Phone: (818) 523 7003
Email: emilychristinedavis@gmail.com

**Comment by justinrporter** [comment]
Nah, you're doing great! It's just kind of complicated so I want to talk about it offline!
